environment-python-2.yml

```diff
@@ -29,12 +29,12 @@ dependencies:
   - jsanimation
   - jupyter
   - jupyter_client
-  - matplotlib
+  - matplotlib <2
   - mpld3
   - mplleaflet
   - nb_conda_kernels
   - nbdime
-  - netcdf4
+  - netcdf4=1.2.4
```

Comment:
Both `netcdf4` and `matplotlib` must be downgraded in order to get latest `iris`. This is pretty bad but `cartopy` does not work with `matplotlib 2` and `iris` PartialDateTime is broken with `netcdf4 > 1.2.4`  :unamused: